src/swagger-specs/query-service.yaml

```diff
@@ -1825,4 +2174,544 @@ definitions:
+          method:
+            type: string
+            description: The HTTP method to use.
+            example: 'PATCH'
+          body:
+            type: string
+            description: The JSON body to send.
+            example: '{ \"op\": \"replace\", \"path\": \"/status\", \"value\": \"enable/disable\" }'
+      get_list_of_subscribers_by_alert_type:
+        type: object
+        description: Useful information including the method and URI to return a list of all subscribers for given assetId and alertType.
+        properties:
+          href:
+            type: string
+            description: Return a list of all subscribers for given assetId and alertType.
+            example: 'https://platform-int.adobe.io/data/foundation/query/alert-subscriptions/c4f67291-1161-4943-bc29-8736469bb928/failure'
+          method:
+            type: string
+            description: The HTTP method to use.
+            example: 'GET'
+      delete:
+        type: object
+        description: Useful information including the method and URI to delete an alert for particular assetId and alertType.
+        properties:
+          href:
+            type: string
+            description: The URI to delete an alert for particular assetId and alertType.
+            example: 'https://platform-int.adobe.io/data/foundation/query/alert-subscriptions/c4f67291-1161-4943-bc29-8736469bb928/failure'
+          method:
+            type: string
+            description: The HTTP method to use.
+            example: 'DELETE'
+
+  alerts_per_imsorg_and_sandbox:
+    type: object
+    properties:
+      alerts:
+        type: array
+        items:
+          properties:
+            id:
+              $ref: '#/definitions/Id'
+            alertType:
+              $ref: '#/definitions/AlertType'
+            status:
+              $ref: '#/definitions/Status'
+            assetId:
+              $ref: '#/definitions/AssetId'
+            _links:
+              $ref: '#/definitions/alert_links'
+      _page:
+        $ref: '#/definitions/PagingProperties'
+      _links:
+        $ref: '#/definitions/query_list_links'
+      version: 
+        $ref: '#/definitions/version'
+
+  alerts_per_imsorg_and_sandbox_response:
+    type: object
+    description: All alerts per organization and sandbox.
+    required:
+      - id
+      - name
+    properties:
+      alerts:
+        $ref: '#/definitions/CoreAlertProperties'
+      _page:
+        $ref: '#/definitions/PagingProperties'
+      _links:
+        $ref: '#/definitions/query_list_links'
+      version: 
+        $ref: '#/definitions/version'
+
+  CoreAlertProperties:
+    type: object
+    description: Contains properties to describe the query ID, alert ID, type, and status of alert.
+    properties:
+      id:
+        type: string
+        description: The alert name that appears in the alerts tab of the Adobe Experience Platform UI.
```

Comment:
Do we need to mention the UI here, considering some will just be using the API? If we do, "Alerts" needs to be capitalized when referencing a UI tab. I'm assuming that we don't though, so I would just describe this as a unique system-generated ID for the alert.